check_pgactivity

```diff
@@ -379,7 +383,20 @@ script relies on the system default temporary directory if possible.
 =item B<-P>, B<--psql> FILE
 
 Path to the C<psql> executable (default: "psql").
-It should be version 8.3 at least, but the server can be older.
+
+Because check_pgactivity uses C<psql -w>, C<psql> must be version 8.4 at least,
+but the server can be older.
+
```

Comment:
useless line IMHO, this is the same subject

Reply:
done in next commit.